apps/demo.namerank.io/app/layout.tsx

```diff
@@ -0,0 +1,34 @@
+import type { Metadata } from "next";
+import { Geist, Geist_Mono } from "next/font/google";
+import "./globals.css";
+
+const geistSans = Geist({
```

Comment:
We can use Inter to match the existing branding.

Reply:
@notrab Yes that would be nice :+1: